jestConfig.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   collectCoverage: true,
   collectCoverageFrom: [
-    'src/**/*.js',
+    'script/**.js',
```

Comment:
Since this package will only ever contain the aggregate translations script, should we just put the script in the lib folder?

Reply:
I like the pattern of having the script in the script directory and leaving `lib` for compiled source code.

Reply:
I like leaving `lib` for the compiled code thought I felt like `script` was a little weird as singular but then it is a single script repo. Could just add an `aggregate-translations` dir and drop `script` 🤷‍♀ but that might feel more confusing if we migrate more scripts to standalone packages.

That being said. Thoughts on moving the supported terra locales config to `config/terraSupportedLocales.js` and/or adding `config/terraI81n.config.js` file with our defaults for quick reference?

Reply:
*Player three has joined*

Since this is now it's own repository, does it make sense to move the scripts into a src directory?

Reply:
I don't see the lib folder exclusively for compiled code so much as exported code. I'd be ok with src too, mainly I'd like this repo to look more like the other oss node modules. Using lib seems to be a pattern? ie 
https://github.com/webpack/webpack-dev-server
https://github.com/babel/babel
https://github.com/expressjs/express

Reply:
Using `lib` sounds good to me. +1 to aligning with other OSS repos.

Reply:
sounds like a plan 👍